spring-boot-docs/src/main/asciidoc/appendix-executable-jar-format.adoc

```diff
@@ -198,7 +198,13 @@ the appropriate launcher:
 
 `PropertiesLauncher` has a few special features that can be enabled with external
 properties (System properties, environment variables, manifest entries or
-`application.properties`).
+`loader.properties`).
+
+NOTE: `PropertiesLauncher` supports loading properties from
+`loader.properties` and also (for historic reasons)
+`application.properties`. To avoid confusion, we recommend using
+`loader.properties` exclusively, and support for
+`application.properties` may be removed in the future.
```

Comment:
I think we should go further and say that support for `application.properties` is deprecated. We can then remove it in 2.0.